tests/cli-exact.rs

```diff
@@ -38,6 +38,7 @@ info: downloading component 'rust-docs'
 info: downloading component 'rust-std'
 info: downloading component 'rustc'
 info: installing component 'cargo'
+Defaulting to 500.0 MiB unpack ram
```

Comment:
Can we guarantee that all our CI systems will permit this, or should we actually set a limit ourselves to ensure this doesn't occur?

Reply:
It passed CI. Any ulimit / job limit we set will by necessity be lower than what CI has available, not higher.

We can't guarantee developers will have 500MB available, or that IDEs won't put ulimits around test runs etc. But if an IDE puts a hard ulimit around a test run, we can't actually raise it ourselves anyway.

Reply:
OK, I guess we know what to fix if it turns out to be a problem in future.  I've got plenty of RAM for running my tests too :D